qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -1,7 +1,8 @@
 from typing import Any, Optional, Dict
 
+from qcodes import Parameter, DelegateParameter
 from qcodes.instrument.base import Instrument
-from qcodes.utils.validators import Enum
+from qcodes.utils.validators import Enum, Numbers
 
 
 class SP983C(Instrument):
```

Comment:
Before you merge can you update the doc string with information about the input_offset_voltage parameter? Thanks!

Reply:
It would probably also be nice to add an example notebook that shows this